packages/flutter_tools/test/general.shard/cache_test.dart

```diff
@@ -8,8 +8,12 @@ import 'package:file_testing/file_testing.dart';
 import 'package:meta/meta.dart';
 import 'package:mockito/mockito.dart';
 import 'package:platform/platform.dart';
+import 'package:process/process.dart';
 
```

Comment:
nit: remove the newlines from the package: import list, and sort.

Reply:
Done